docs/features.md

```diff
@@ -0,0 +1,18 @@
+# Supported Kubernetes features
+
+This is an uncomplete list of Kubernetes features supported by rktlet.
+
+| Feature             | Supported | Comments |
+| ------------------- |:---------:| -------- |
+| Pod lifecycle       | YES       |          |
+| Container lifecycle | YES       |          |
+| Logging             | YES       |          |
+| Attach              | NO        | See [#8](https://github.com/kubernetes-incubator/rktlet/issues/8) |
+| Port forwarding     | NO        |          |
```

Comment:
is it only for `kubectl port-forward`?
Or also pods with a `hostPort`?

Reply:
only `kubectl port-forward`

Reply:
I can write `kubectl port-forward` instead if that's clearer